pipeline-scripts/release.groovy

```diff
@@ -616,6 +616,9 @@ def openCincinnatiPRs(releaseName, advisory, candidate_only = false,ghorg = 'ope
             if ( major == 4 && minor == 1 ) {
                 prefixes = [ "prerelease", "stable"]
             }
+            if ( major == 4 && minor >= 6 ) {
```

Comment:
From what I've been able to gather, we need to have EUS channels for 4.6 (and later, 4.10), but not for 4.7, 4.8, ....

Reply:
may I suggest that we create something like `eusVersions = [...]` or a function `def isEUS(...)` in buildlib or commonlib, and use that to make comparisons instead?

Reply:
What I understand is that currently only for 4.6 update eus channel with the same content we updated in stable channel.

Reply:
Yes. I think a `commonlib.eusVersions` next to `commonlib.ocp4Versions` would be perfect.